README.md

```diff
@@ -10,4 +10,20 @@
 - Works with following scopes (some may not be needed): users:read, commands, chat:write, channels:read, channels:manage, channels:join, channels:history
 - Interactivity should be enabled with redirect going to /interactive
 - Make a slash command (arbitary name) with request URL going to /commands
-- Add /events URL to events subscriptions and enable
\ No newline at end of file
+- Add /events URL to events subscriptions and enable
+
+## Environment Variables
+You will need a .env file with the following
+```
+SLACK_SIGNING_SECRET=
+SLACK_TOKEN=
+
+
+JIRA_USERNAME=
+JIRA_PASSWORD=
+JIRA_SERVER=
+JIRA_PROJECT_KEY=TEST111
+CREATE_JIRA_VERSIONS=False
```

Comment:
Would probably suggest `ENABLE_JIRA_INTEGRATION` or `ENABLE_CREATE_JIRA_VERSIONS` as the toggle name so it's clear it's a switch

Reply:
I started writing the exact same one then decided not to for now heh. I will add it :)